tests/Bridge/Symfony/Routing/ApiLoaderTest.php

```diff
@@ -113,6 +134,24 @@ private function getApiLoaderWithResourceMetadata(ResourceMetadata $resourceMeta
 
         $kernelProphecy = $this->prophesize(KernelInterface::class);
         $kernelProphecy->locateResource(Argument::any())->willReturn($routingConfig);
+        $possibleArguments = [
+            'api_platform.action.get_collection',
+            'api_platform.action.post_collection',
+            'api_platform.action.get_item',
+            'api_platform.action.put_item',
+            'api_platform.action.delete_item',
+        ];
+
+        $containerInterfaceProphecy = $this->prophesize(ContainerInterface::class);
+        $containerInterfaceProphecy->reveal();
```

Comment:
>"I don't know PHPSpec so I let you write tests." or "I don't know PHPSpec so I wrote PHPUnit tests."

The problem isn't here, it's "I won't contribute because I'm not willing to learn a new testing tool just for this project". It's arguable, but that's the reason why we stayed to phpunit.